cmd/hook.go

```diff
@@ -314,7 +314,7 @@ func runHookPostReceive(c *cli.Context) error {
 	}
 
 	// Now if we're an internal don't do anything else
-	if os.Getenv(models.EnvIsInternal) == "true" {
+	if os.Getenv(repo_module.EnvIsInternal) == "true" {
```

Comment:
Again, wouldn't it be better to use
```suggestion
	if isInternal, _ := strconv.ParseBool(os.Getenv(repo_module.EnvIsInternal)); isInternal {
```